storage/simulation/cluster.go

```diff
@@ -34,9 +34,9 @@ import (
 	"github.com/cockroachdb/cockroach/util/stop"
 )
 
-// Cluster maintains a list of all nodes, stores and ranges as well as any
+// simCluster maintains a list of all nodes, stores and ranges as well as any
 // shared resources.
-type Cluster struct {
+type simCluster struct {
```

Comment:
I had to add the sim prefixes to avoid collisions with variable names, but you're right, I'll revert changes to this package.